### PR TITLE
Dev/piano roll layout

### PR DESCRIPTION
**Pull request by satoshi-szk**

### CODE REVIEW

**Comment by satoshi-szk** [comment]
Hi, @zigen.
I made piano roll prototype which can render notes, but does not provide note editing.
Could you review my codes?

**Comment by satoshi-szk** [comment]
Hi @zigen, I've refactored piano roll as a Vue component.
If OK, please approve this PR.

**Comment by zigen** [comment]
how fast!!!!!
let me check it.

File: src/renderer/modules/pianoRoll/utils.js [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+};
+/**
+ * @param {string} className
+ * @param {string} [id]
+ * @returns {HTMLDivElement}
+ */
+export const initDivElement = (className, id) => {
+  const elt = document.createElement('div');
+  elt.classList.add(paramCase(className));
+  elt.id = id || paramCase(className);
```
Comment (zigen) [inline src/renderer/modules/pianoRoll/utils.js]: this seems misleading. if `id` isn't specified, fill it with className?

**Comment by satoshi-szk** [comment]
Thank you😊

**Review by zigen — APPROVED** [review]
I didn't test, but seems almost good

**Comment by satoshi-szk** [comment]
You still have the Error on `y dev`?

**Pull request merged**